jetbrains-core/tst/software/aws/toolkits/jetbrains/services/sqs/SqsServiceNodeTest.kt

```diff
@@ -5,40 +5,41 @@ package software.aws.toolkits.jetbrains.services.sqs
 
 import com.intellij.testFramework.ProjectRule
 import org.assertj.core.api.Assertions.assertThat
-import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
-import software.aws.toolkits.jetbrains.core.MockResourceCache
+import software.aws.toolkits.jetbrains.core.MockResourceCacheRule
 import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerEmptyNode
 import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerErrorNode
 import software.aws.toolkits.jetbrains.core.explorer.nodes.SqsExplorerRootNode
 import software.aws.toolkits.jetbrains.services.sqs.resources.SqsResources
-import java.util.concurrent.CompletableFuture
+import software.amazon.awssdk.utils.CompletableFutureUtils
 
 class SqsServiceNodeTest {
     @JvmField
     @Rule
     val projectRule = ProjectRule()
 
-    @Before
-    fun setUp() {
-        resourceCache().clear()
-    }
+    @JvmField
+    @Rule
+    val resourceCache = MockResourceCacheRule(projectRule)
 
     @Test
-    fun sqsQueuesAreListed() {
+    fun `Sqs queues are listed`() {
         val queueList = listOf(
```

Comment:
nit: since this is never reused, why give it a name? Could just put it into the resource cache call